packages/frontend/src/presentation/Navigator.tsx

```diff
@@ -26,7 +26,7 @@ const Stack = createStackNavigator<RootParamList>();
 export function Navigator(): React.ReactElement {
   return (
     <NavigationNativeContainer>
-      <Stack.Navigator initialRouteName="CardSearchFilter">
+      <Stack.Navigator initialRouteName="Welcome">
```

Comment:
Seria melhor se isso fosse um Enumeravel (enum) ao inves de usar strings soltas.
```suggestion
      <Stack.Navigator initialRouteName={SCREENS.WELCOME}>
```